src/Uno.UI.Composition/Composition/CompositionAnimation.cs

```diff
@@ -23,7 +23,9 @@ internal CompositionAnimation(Compositor compositor) : base(compositor)
 	internal Dictionary<string, Vector2> Vector2Parameters { get; } = new();
 	internal Dictionary<string, Vector3> Vector3Parameters { get; } = new();
 
-	internal event Action<CompositionAnimation>? PropertyChanged;
+	internal event Action<CompositionAnimation>? AnimationFrame;
```

Comment:
Was not able to find where this event is being listen?

Reply:
Oh yes it's here https://github.com/unoplatform/uno/pull/16465/files#diff-e6d71552b4115f635edc831930106261efbe09bd0ec1fc38485327ddadfbf3c4R90